src/MessageHandler/Order/SendRemotePushNotification.php

```diff
@@ -96,13 +94,16 @@ public function __invoke(OrderCreated|OrderAccepted|OrderStateChanged $event)
                 ],
             ];
 
-            $users = array_map(function ($user) {
-                return $user->getUsername();
-            }, $owners);
-            $users = array_unique($owners);
+            $users = array_reduce(array_unique($owners), function ($carry, $owner) {
```

Comment:
I don't think this is needed. It seems that`$owners` is already an array of User objects

Reply:
Yepp you r right, I can confirm that on `LocalBusiness` class:
```
    public function addOwner(User $owner)
    {
        $owner->addRestaurant($this);
        $this->owners->add($owner);
    }
```
And on `Order` class:
```
    public function getNotificationRecipients(): Collection
    {
        $recipients = new ArrayCollection();

        foreach ($this->getRestaurants() as $restaurant) {
            foreach ($restaurant->getOwners() as $owner) {
                $recipients->add($owner);
            }
        }

        return $recipients;
    }
```

Reply:
Done at https://github.com/coopcycle/coopcycle-web/pull/5062/commits/a619c6a22132cd14bf0e31cb2ce1977c81f1c734